### PR TITLE
Fix markdown link to Nginx home page

### PR DESCRIPTION
**Pull request by jen-huang**
## What does this PR do?

This PR fixes a markdown link typo in the nginx package description.

### CODE REVIEW

File: packages/nginx/docs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Nginx Integration
 
-This integration periodically fetches metrics from [https://nginx.org/](Nginx) servers. It can parse access and error
+This integration periodically fetches metrics from [Nginx](https://nginx.org/) servers. It can parse access and error
```
Comment (ph) [inline packages/nginx/docs/README.md:3]: :facepalm:  I've seriously thought it was good markdown. I am sorry @jen-huang

Comment (jen-huang) [reply packages/nginx/docs/README.md:3]: no worries, it was sneaky :D

Comment (ph) [reply packages/nginx/docs/README.md:3]: :see_no_evil:

**Comment by ph** [comment]
@mtojek Should we take care of deploying the packaging or you will do it on your side?

**Pull request merged**